src/patternfly/layouts/Page/styles.scss

```diff
@@ -4,97 +4,86 @@
-  --pf-l-page__header-brand-logo--Height:                   pf-size-prem(11px);
+  --pf-l-page__header-brand-logo--Height: pf-size-prem(11px);
 
   // Header context selector
-  --pf-l-page__header-selector--PaddingLeft:                var(--pf-global--spacer--md);
-  --pf-l-page__header-selector--PaddingTop:                 var(--pf-global--spacer--lg);
-  --pf-l-page__header-selector--PaddingBottom:              var(--pf-global--spacer--lg);
-  --pf-l-page__header-selector--PaddingLeft:            var(--pf-global--spacer--xl);
+  --pf-l-page__header-selector--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-l-page__header-selector--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-l-page__header-selector--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-l-page__header-selector--PaddingLeft: var(--pf-global--spacer--xl);
 
   // Header nav
-  --pf-l-page__header-nav--PaddingRight:                    var(--pf-global--spacer--xl);
-  --pf-l-page__header-nav--PaddingLeft:                     var(--pf-global--spacer--xl);
-  --pf-l-page__header-nav--MarginTop--sm:                   var(--pf-global--spacer--md);
-  --pf-l-page__header-nav--MarginLeft--md:                  var(--pf-global--spacer--xl);
-  --pf-l-page__header-nav--MarginRight--md:                 var(--pf-global--spacer--xl);
-  --pf-l-page__header-nav--BackgroundColor--sm:             var(--pf-global--BackgroundColor--dark-transparent-100);
-
-  // Header toolbar
-  --pf-l-page__header-toolbar--PaddingTop:                  var(--pf-global--spacer--xl);
-  --pf-l-page__header-toolbar--PaddingRight:                var(--pf-global--spacer--xl);
-  --pf-l-page__header-toolbar--PaddingBottom:               var(--pf-global--spacer--md);
-  --pf-l-page__header-tools--toolbar--MarginTop:            var(--pf-global--spacer--md);
-  --pf-l-page__header-tools--toolbar--MarginBottom:         var(--pf-global--spacer--md);
-  --pf-l-page__header-tools--MarginRight:                   var(--pf-global--spacer--xl);
-
-  // Header avatar
-  --pf-c-avatar--MarginLeft:                                var(--pf-global--spacer--md);
+  --pf-l-page__header-nav--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-l-page__header-nav--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-l-page__header-nav--MarginTop--sm: var(--pf-global--spacer--md);
+  --pf-l-page__header-nav--MarginLeft--md: var(--pf-global--spacer--xl);
+  --pf-l-page__header-nav--MarginRight--md: var(--pf-global--spacer--xl);
+  --pf-l-page__header-nav--BackgroundColor--sm: var(--pf-global--BackgroundColor--dark-transparent-100);
+
+  // Header tools
+  --pf-l-page__header-tools--MarginRight: var(--pf-global--spacer--xl);
+  --pf-l-page__header-tools--toolbar--MarginTop: var(--pf-global--spacer--md);
+  --pf-l-page__header-tools--toolbar--MarginBottom: var(--pf-global--spacer--md);
+  --pf-l-page__header-tools--avatar--MarginLeft: var(--pf-global--spacer--md);
+  --pf-l-page__header-tools--avatar--Width: pf-size-prem(48px);
```

Comment:
Why is this hardcoded? can we add a comment to explain?

Reply:
updated